### PR TITLE
feat(analytics): Add Analytics for #75

### PR DESCRIPTION
**Pull request by simonsmh**
Please be sure to put the `canary` branch as the target for your pull request.

**What kind of change does this PR introduce?** (check one with "x")

- [x] Bug fix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -145,6 +145,14 @@ search:
     use: google
     swiftype_key:
 
+# Analytics Systems
+# Available value:
+#     baidu | google
+analytics:
+    use: 
+    baidu_id: "12345678901234567890123456789012"
```
Comment (viosey) [inline _config.yml]: value of baidu_id & google_id should be null

File: _config.yml [hunk 2]
```diff
@@ -145,6 +145,14 @@ search:
     use: google
     swiftype_key:
 
+# Analytics Systems
+# Available value:
+#     baidu | google
+analytics:
```
Comment (viosey) [inline _config.yml]: delete `use` field.
set null value of `baidu_id` & `google_id`

File: layout/_partial/head.ejs [hunk 3]
```diff
@@ -95,6 +95,29 @@
         }
     </script>
 
+    <% if(theme.analytics.use === 'baidu') { %>
```
Comment (viosey) [inline layout/_partial/head.ejs]: change this judgement.

Template: 
```
<% if(theme.analytics.baidu_id){ %>
    baidu analytics code
<% } %>
```

**Comment by simonsmh** [comment]
Done. Please review. Thx

**Comment by viosey** [comment]
Alright, [markdown docs](https://github.com/viosey/material-theme-docs)
this is the docs, you can fork it, and PR if you will.

**Pull request merged**